tox.ini

```diff
@@ -13,6 +13,7 @@ setenv =
     PYTHONUNBUFFERED = 1
     OBJC_DISABLE_INITIALIZE_FORK_SAFETY = YES
     dev: DEV = {env:DEV:true}
+    dev,tests: PYTHONDEVMODE = {env:PYTHONDEVMODE:1}
```

Comment:
Not sure what the right scope for this is. `dev` and `tests`? 

Or all of them, ie, with no scope:

`PYTHONDEVMODE = {env:PYTHONDEVMODE:1}`

Reply:
In `make dev` we get a bunch of


```web-https (stderr)   | /home/marcos/hypo/lms/.tox/dev/lib/python3.8/site-packages/jsonschema/_validators.py:321: ResourceWarning: unclosed <ssl.SSLSocket fd=11, family=AddressFamily.AF_INET, type=SocketKind.SOCK_STREAM, proto=6, laddr=('192.168.1.22', 55836), raddr=('34.232.82.173', 443)>
```

with this.

Maybe we should use it only for the tests then?